htdocs/product/class/api_products.class.php

```diff
@@ -1371,18 +1372,13 @@ public function addVariant($id, $weight_impact, $price_impact, $price_impact_is_
         }
 
         $prodcomb = new ProductCombination($this->db);
-        if (! $prodcomb->fetchByProductCombination2ValuePairs($id, $features))
```

Comment:
What is goal of removing this ?

Reply:
C'est moi qui ait créé ce morceau de code. Je me suis rendu compte que c'est redondant, car createProductCombination effectue déjà cette vérification. De plus, un comportement est prévu au cas où la déclinaison existe déjà. Ce qui fait que ce morceau de code bride totalement la fonction createProductCombination.

Reply:
Mince, je crois que j'ai cliquer sur un truc qui fallait pas. "Merge branch 'develop' into patch-3"